### PR TITLE
RDY: elastic

### PR DESCRIPTION
**Pull request by jochym**

### CODE REVIEW

File: recipes/elastic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py3k and not win]
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py<34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
```
Comment (scopatz) [inline recipes/elastic/meta.yaml]: Please remove these comments

**Comment by scopatz** [comment]
LGTM, just the one small issue.

**Comment by jochym** [comment]
Thanks @scopatz for the review. The package was waiting on dependency package (spglib). I'll fix it now and finish it up.

File: recipes/elastic/meta.yaml (hunk 1 above)
Comment (jochym) [inline recipes/elastic/meta.yaml]: Done

**Comment by scopatz** [comment]
Ok, please ping when it is ready!

**Comment by jochym** [comment]
@scopatz pinging since elastic recipe is finally ready

**Comment by scopatz** [comment]
Thanks!

**Comment by jochym** [comment]
Something went wrong with merge build @scopatz . Would you take a look? I have no access to the build.

**Comment by scopatz** [comment]
It seems like a feedstock was created, so please just fix and rebuild by putting PRs into that repo.

**Comment by jochym** [comment]
Yes. It seems all is fine after all. Thanks.

**Pull request merged**